browsers/samsunginternet_android.json

```diff
@@ -273,9 +273,20 @@
         },
         "23.0": {
           "release_date": "2023-10-18",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "115"
+        },
+        "24.0": {
+          "release_date": "2024-01-25",
```

Comment:
Very rough guess based on the date the APK for v24 was uploaded to apk mirror (see https://www.apkmirror.com/apk/samsung-electronics-co-ltd/samsung-internet-for-android/samsung-internet-for-android-24-0-0-47-release/)

Reply:
Thanks for the link. iIt seems that version 25 was released yesterday?

Reply:
Yep looks like it! I'll update this PR